### DISCUSSION TITLE
Articles report / Payments report

### DISCUSSION THREAD

**Discussion by Kx29PM** [btcpayserver/btcpayserver#4598] [Ideas / Feature Requests]
Hello,

it would be great if in the backend you could sort by articles or payments. And also if you could choose a specific time period. Personally, I need to do the inventory/payments check at the end of each month (sometimes daily), and it's easier if I see it all on one page. They are also divided by currency. 

Those are screenshots from the General Bytes PoS that we currently use.

<img width="1268" alt="article report" src="https://user-images.githubusercontent.com/124400297/216705993-c2fbd2ca-84e5-4552-a25a-941ab5f52fe2.png">

<img width="1272" alt="payments report" src="https://user-images.githubusercontent.com/124400297/216705998-0a6b6aea-2519-4c50-a8f5-de9d9cbb49ea.png">

**Comment by pavlenex** [comment]
This one was implemented.
